CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Change Log
 
+### v4.1.0 (2016/2/20 12:04 +07:00)
```

Comment:
`yargs` is already at `4.1.0`, https://github.com/yargs/yargs/commit/3808fabcd36645a1c855385e62c31fd5917ee447